pkgs/development/libraries/libcint/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, openblas
+, python
```

Comment:
Should this maybe be `python3` instead? `python` still points to `python2`.

Reply:
It builds with py2.7-numpy-1.16.5, and this is just for a test input.

Actually, on closer look the tests actually require python2 due to their print statement handling. https://github.com/sunqm/libcint/blob/c9b675d681cd50bea5e3b1b52867b4b6ee2504de/testsuite/test_cint.py#L163

Reply:
I was just about to ask you to note that in a comment, but I see you already did. Great :)